css/templates/themes/1/page.css

```diff
@@ -0,0 +1,139 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../core';
+@import '../../../../components/byline/byline';
+@import '../../../../components/dropdown/dropdown';
+@import '../../../../components/footer/footer';
+@import '../../../../components/images/fullpage-hero';
+@import '../../../../components/images/images';
+@import '../../../../components/inputs/inputs';
+@import '../../../../components/navbar/navbar';
+@import '../../../../components/related-article/related-article';
+@import '../../../../components/related-section/related-section';
+@import '../../../../components/socialbox/socialbox';
+@import '../../../../components/social-follow/social-follow';
+@import 'page-vars';
+
+h1,
+.h1 {
+  letter-spacing: 0;
+}
+
+h2,
+.h2 {
+  letter-spacing: 0;
+}
+
+h3,
+.h3 {
+  color: var(--color-black);
+  font-family: var(--font-secondary);
+  letter-spacing: .6px;
+}
+
+h4,
+.h4 {
+  letter-spacing: .2px;
+  line-height: var(--line-height-2);
+}
+
+.ampstart-navbar-trigger {
+  font-size: 1.5rem;
+}
+
+.ampstart-nav-dropdown .ampstart-dropdown-item,
+.ampstart-nav-dropdown .ampstart-dropdown>section>header,
+.ampstart-sidebar-faq {
+  background: transparent;
+  color: var(--color-white);
+  font-family: var(--font-secondary);
+}
+
+.ampstart-nav-dropdown amp-accordion ul {
+  background: transparent;
+  margin-left: 1rem;
+}
+
+.ampstart-sidebar .ampstart-dropdown-item,
+.ampstart-sidebar .ampstart-dropdown header,
+.ampstart-sidebar .ampstart-faq-item {
+  margin-bottom: 1.5rem;
+}
+
+.ampstart-dropdown>section>header:after {
+  color: var(--color-white);
+}
+
+.ampstart-sidebar .ampstart-icon {
+  fill: var(--color-white);
+}
+
+.ampstart-image-fullpage-hero figcaption,
+.ampstart-article-summary header {
+  background: var(--color-primary);
+  opacity: .9;
+}
+
+.ampstart-fullpage-hero-heading {
+  line-height: 1.6;
+}
+
+.ampstart-image-fullpage-hero .ampstart-image-credit {
+  background: none;
+  padding: 0;
+}
+
+.ampstart-fullpage-hero-heading-text {
+  background: none;
+  padding: 0;
+}
+
+.ampstart-readmore {
+  background: none;
+  display: block;
+  margin: 0;
+  padding: var(--space-2) 0 var(--space-4);
+}
+
+.ampstart-readmore::after {
+  content: " ";
+  border: 2px solid #000;
+  border-width: 0 2px 2px 0;
+  bottom: 2rem;
+  display: block;
+  height: 10px;
+  left: calc(50% - 5px);
+  position: absolute;
+  transform: rotateZ(45deg);
+  width: 10px;
+}
+
+.ampstart-sidebar {
+  background-color: var(--color-black);
```

Comment:
Both `headerbar` and `sidebar` user `--color-navbar-primary`, but this design has a white `headerbar` and a black sidebar, so one has to be overridden. I've changed this now the the `headerbar` as it's simpler, and used the variables where appropriate.